specs/models/neo_artifact_spec.py

```diff
@@ -0,0 +1,90 @@
+from mamba import description, before, after, it, context
+from expects import expect, equal
+from application.models.neo_artifact import NeoArtifact
+from application.models.neo_tag import NeoTag
+from specs.spec_helpers import Context
+from specs.models.custom_matcher import be_uuid, have_node, have_relationship
+from flask import current_app
+from py2neo import Node
+
+node_name = "NeoArtifact"
+with description('NeoArtifact') as self:
```

Comment:
Tests for the NeoArtifact. I think I haven't tested everything that the OGM provides. Not sure if we should have those though as they are provided by the base class?